recipes/descent/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "descent" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SimonBoothroyd/descent/archive/{{ version }}.tar.gz
+  sha256: bd09edab42408891be4d7460cb79470fb770dc0981259f07dd697821e587e99d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - pip
+    - versioningit
+  run:
+    - python >=3.10
+    - smee >=0.4.0  # [unix]
+    - pydantic >=2
+    - pyarrow
+    - scipy
+    - pytorch  # [unix]
```

Comment:
Unless smee and pytorch are packages that won't ever be available on Windows you don't need this. Make a simple noarch and let the solver block the Windows installation until the missing dependencies are available.